app/slave/subjob_executor.py

```diff
@@ -82,6 +84,8 @@ def execute_subjob(self, build_id, subjob_id, subjob_artifact_dir, atomic_comman
                 'ARTIFACT_DIR': atom_artifact_dir,
                 'ATOM_ID': atom_id,
                 'EXECUTOR_INDEX': self.id,
```

Comment:
yes.  I'll make a comment in the code, and then i should change any documentation examples.